src/libs/ReportUtils.js

```diff
@@ -1653,13 +1653,13 @@ function buildOptimisticTaskCommentReportAction(taskReportID, taskTitle, taskAss
  * @param {String} chatReportID - Report ID of the chat where the IOU is.
  * @param {String} currency - IOU currency.
  * @param {Boolean} isSendingMoney - If we send money the IOU should be created as settled
- *
+ * @param {String} unknownPayerEmail - The mail or phone of the payer, usefull when we don't know the login of the user
  * @returns {Object}
  */
-function buildOptimisticIOUReport(payeeAccountID, payerAccountID, total, chatReportID, currency, isSendingMoney = false) {
+function buildOptimisticIOUReport(payeeAccountID, payerAccountID, total, chatReportID, currency, isSendingMoney = false, unknownPayerEmail = '') {
```

Comment:
Lets just call it payerEmail, but in the docs explain the situation
```suggestion
function buildOptimisticIOUReport(payeeAccountID, payerAccountID, total, chatReportID, currency, isSendingMoney = false, payerEmail = '') {
```

Reply:
There is already a payerEmail variable in the function, checking if we need it now